pkg/logger/logger.go

```diff
@@ -440,20 +460,23 @@ func (l *Logger) ReconcileSuccessStream(
 		os.FileMode(utils.DefaultFilePermissions),
 	)
 	if err != nil {
-		return fmt.Errorf("failed to open file %s: %w", path.Join(l.logDir, reconcileSuccessStreamFile), err)
+		err = fmt.Errorf("failed to open file %s: %w", path.Join(l.logDir, reconcileSuccessStreamFile), err)
+		l.zapLogger.Error(err.Error())
+		return err
 	}
 
 	defer closeFile(f)
 
 	log.Printf(
```

Comment:
This is still using stdlib log not zap log. We should change it.

Reply:
@shrimalmadhur the line 464 is using zapplogger 
here is an example from other similar part, which before change color
`{"level":"\u001b[31mERROR\u001b[0m","ts":1670542864.602814,"caller":"logger/logger.go:416","msg":"failed to open file cli-data/check-data/97499d288cb979c554f55ccdd1430b0a449b5ff4ff4ced0cdf87ab7eb9cf619e/balance_changes.txt: open cli-data/check-data/97499d288cb979c554f55ccdd1430b0a449b5ff4ff4ced0cdf87ab7eb9cf619e/balance_changes.txt: no such file or directory","blockchain":"arbitrum","network":"mainnet","requestUUID":"jingarbitrumssrrrrr","check_type":"data","stacktrace":"github.com/coinbase/rosetta-cli/pkg/logger.`